sovrin_client/test/cli/test_agent_cli.py

```diff
@@ -0,0 +1,120 @@
+import pytest
+
+from functools import partial
+
+from plenum.common.eventually import eventually
+from plenum.test.cli.helper import TestCliCore
+from plenum.test.testable import Spyable
+from sovrin_client.agent.agent_cli import AgentCli
+from sovrin_client.test.agent.acme import createAcme
+from sovrin_client.test.cli.helper import getCliBuilder
+from sovrin_client.test.cli.test_tutorial import acmeWithEndpointAdded,\
+    philCli, acmeAddedByPhil, connectIfNotAlreadyConnected
+
+
+@Spyable(methods=[AgentCli.print, AgentCli.printTokens])
+class TestAgentCLI(AgentCli, TestCliCore):
+    pass
+
+
+@pytest.fixture(scope='module')
+def agentCliBuilder(tdir, tdirWithPoolTxns, tdirWithDomainTxns, tconf, cliTempLogger):
+    return partial(getCliBuilder, tdir=tdir, tconf=tconf,
+                   tdirWithPoolTxns=tdirWithPoolTxns,
+                   tdirWithDomainTxns=tdirWithDomainTxns,
+                   logFileName=cliTempLogger, cliClass=TestAgentCLI)
+
+
+@pytest.fixture(scope='module')
+def acmeAgentCli(agentCliBuilder, acmeAgentPort):
+    yield from agentCliBuilder(
+        name='Acme-Agent',
+        agentCreator=lambda: createAcme(port=acmeAgentPort)
+    )('Acme-Agent')
+
+
+@pytest.fixture(scope='module')
+def acmeAgentCliRunning(acmeWithEndpointAdded, acmeAgentCli, looper):
+    looper.run(acmeAgentCli.agent.bootstrap())
+    return acmeAgentCli
+
+
+def test_acme_cli_started_successfully(be, acmeAgentCliRunning):
+    be(acmeAgentCliRunning)
+    assert acmeAgentCliRunning.currPromptText == 'Acme-Agent'
+
+
+def sendProofRequest(be, do, agentCli, userMap):
+    be(agentCli)
+    do('send proof-request {pr-name} to {send-proof-target}',
+       within=5,
+       mapper=userMap,
+       expect=[
+           'Sent proof request {pr-name} to {send-proof-target}'
+       ])
+
```

Comment:
If target is expected to receive this message, then I would suggest, assert the expected message on target's cli as well as below (for that you'll have to pass target's cli to this function, in this case it might be aliceCLI)

be(aliceCLI)
do(None, within=5, mapper=userMap, expect=['Proof request {pr-name} received.'])